keps/sig-api-machinery/20190226-network-proxy.md

```diff
@@ -425,6 +426,19 @@ Alpha:
 - Kubernetes will not ship with a network proxy. The feature will work with the sample network proxy in https://github.com/kubernetes-sigs/apiserver-network-proxy
 - Demonstrate that the API Server Network Proxy eliminates the need for the SSH Tunnels.
 
+Beta:
+
+- All [Kube API Server egress points](#kubernetes-api-server-outbound-requests) have been implemented to use the
+  EgressSelector.
+- Have official releases of the [Konnectivity Server and Agent](https://github.com/kubernetes-sigs/apiserver-network-proxy) reference implementations.
+- Have at least one OSS kube-up implementation where the feature can be turned on and
+  demonstrated.
+- Have run a basic load test with egresses enabled through the Konnectivity
```

Comment:
I don't see a requirement for metrics. How will you evaluate how well we performed?